hibernate-envers/src/main/java/org/hibernate/envers/boot/internal/LegacyModifiedColumnNamingStrategy.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.envers.boot.internal;
+
+import org.hibernate.envers.boot.spi.ModifiedColumnNamingStrategy;
+import org.hibernate.envers.configuration.internal.GlobalConfiguration;
+import org.hibernate.envers.configuration.internal.metadata.MetadataTools;
+import org.hibernate.envers.configuration.internal.metadata.reader.PropertyAuditingData;
+import org.hibernate.mapping.Value;
+
+import org.dom4j.Element;
+
+/**
+ * A {@link ModifiedColumnNamingStrategy} that adds modified columns with the following rules:
+ * <ul>
+ *     <li>If an audit annotation modified column name is supplied, use it directly with no suffix.</li>
+ *     <li>If no audit annotation modified column name is present, use the property name appended with suffix.</li>
+ * </ul>
+ *
+ * This is the default Envers modified column naming behavior.
+ *
+ * @author Chris Cranford
+ * @since 5.4.6
+ */
+public class LegacyModifiedColumnNamingStrategy implements ModifiedColumnNamingStrategy {
+	@Override
+	public void addModifiedColumns(
+			GlobalConfiguration globalCfg,
+			Value value,
+			Element parent,
+			PropertyAuditingData propertyAuditingData) {
+		String columnName = propertyAuditingData.getModifiedFlagName();
+		if ( propertyAuditingData.isModifiedFlagNameExplicitlySpecified() ) {
+			columnName = propertyAuditingData.getExplicitModifiedFlagName();
+		}
```

Comment:
it is not this changing the legacy behaviour? it seems that before we only passed `propertyAuditingData.getModifiedFlagName()`.